### PR TITLE
Added image table detector annotator

### PR DESCRIPTION
**Pull request by gadde5300**
Hi @C-K-Loan ,
I have disabled the light pipeline in this pr as it wasnt working with it enabled.

### CODE REVIEW

**Comment by gadde5300** [comment]
Added a new light pipeline feature to the component universe.

File: nlu/pipe/utils/predict_helper.py [hunk 1]
```diff
@@ -311,7 +311,8 @@ def __predict__(pipe, data, output_level, positions, keep_stranger_features, met
         else:
             pipe.fit()
 
-        pipe.__configure_light_pipe_usage__(DataConversionUtils.size_of(data), multithread)
+        if isinstance(pipe.is_light_pipe_incompatible,()):
+            pipe.__configure_light_pipe_usage__(DataConversionUtils.size_of(data), multithread)
```
Comment (C-K-Loan) [inline nlu/pipe/utils/predict_helper.py:315]: I think this should be if pipe.is_light_pipe_incompatible  and not isinstance()

File: nlu/pipe/pipeline.py [hunk 2]
```diff
@@ -203,7 +203,8 @@ def fit(self, dataset=None, dataset_path=None, label_seperator=','):
                 logger.info(
                     'Fitting on empty Dataframe, could not infer correct training method. This is intended for non-trainable pipelines.')
                 self.vanilla_transformer_pipe = self.spark_estimator_pipe.fit(self.get_sample_spark_dataframe())
-                self.light_transformer_pipe = LightPipeline(self.vanilla_transformer_pipe)
+                if isinstance(self.is_light_pipe_incompatible,()):
```
Comment (C-K-Loan) [inline nlu/pipe/pipeline.py:206]: if self.is_light_pipe_incompatible instead of instance

File: nlu/universe/feature_node_universes.py [hunk 3]
```diff
@@ -297,14 +297,20 @@ class OCR_FEATURE_NODES:
...
-        A.IMAGE_TABLE_CELL_DETECTOR: OcrFeatureNode(A.IMAGE_TABLE_CELL_DETECTOR, [F.OCR_IMAGE, ], [F.OCR_TABLE_CELLS]),
+        A.IMAGE_TABLE_DETECTOR: OcrFeatureNode(A.IMAGE_TABLE_DETECTOR, [F.OCR_IMAGE], [F.OCR_REGION]),
         # TODO REGION or TABLE??? IS IT THE SAME???
-        A.IMAGE_TABLE_CELL2TEXT_TABLE: OcrFeatureNode(A.IMAGE_TABLE_CELL2TEXT_TABLE, [F.OCR_IMAGE, F.OCR_TABLE_CELLS],
+        A.IMAGE_TABLE_CELL_DETECTOR: OcrFeatureNode(A.IMAGE_TABLE_CELL_DETECTOR, [F.IMG_SPLIT_REGIONS, F.OCR_REGION],
+                                                    [F.OCR_TABLE_CELLS]),
+        A.IMAGE_TABLE_CELL2TEXT_TABLE: OcrFeatureNode(A.IMAGE_TABLE_CELL2TEXT_TABLE, [F.IMG_SPLIT_REGIONS,
+                                                                                      F.OCR_TABLE_CELLS],
                                                       [F.OCR_TABLE]),
         # TODO OUPUT!! REGION or TABLE??? IS IT THE SAME???
```
Comment (C-K-Loan) [inline nlu/universe/feature_node_universes.py:315]: Remove commments

File: nlu/pipe/extractors/extractor_methods/ocr_extractors.py [hunk 4]
```diff
@@ -14,7 +14,7 @@ def extract_table(df):
     return exploded_results.toPandas()
 
 
-def extract_tables(df, rename_cols=True):
+def extract_tables(df, rename_cols=False):
     df = df.withColumn("table_index", df.ocr_table.area.index)
```
Comment (C-K-Loan) [inline nlu/pipe/extractors/extractor_methods/ocr_extractors.py:18]: Can you check if this has any effect on the existing table extractor?

**Review by C-K-Loan — CHANGES_REQUESTED** [review]

**Pull request merged**